test/unit/sql/enum.test.js

```diff
@@ -45,9 +62,9 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
           });
         });
 
-        it('uses the default schema if options given', function () {
+        it('uses the default schema if no options given', function () {
             expectsql(sql.pgListEnums(), {
-            postgres: 'SELECT t.typname enum_name, array_agg(e.enumlabel) enum_value FROM pg_type t JOIN pg_enum e ON t.oid = e.enumtypid JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace WHERE n.nspname = \'public\' GROUP BY 1'
+              postgres: 'SELECT t.typname enum_name, array_agg(e.enumlabel) enum_value FROM pg_type t JOIN pg_enum e ON t.oid = e.enumtypid JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace WHERE n.nspname = \'public\' GROUP BY 1'
           });
```

Comment:
I see you fixed the indentation here, looks like it was `expectsql` that needed to be moved back tho :)